go/cluster_management/internal/util/find_cluster_test.go

```diff
@@ -0,0 +1,136 @@
+package util
```

Comment:
Is this test executed anywhere? It looks like it's dependent on real resources in your account, do we need this?